modules/32_power/IntC/equations.gms

```diff
@@ -278,9 +278,12 @@ q32_flexPriceBalance(t,regi)$(cm_FlexTaxFeedback eq 1)..
 *** while inflexible technologies are penalized (v32_flexPriceShare > 1).  
 *** Flexibility tax is switched only if cm_flex_tax = 1 and is active from 2025 onwards. 
```

Comment:
Good thing you deleted the
> - [ ] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date

part from the template.

Still better would have been to explain why this is limited to 2, and how that relates to the 200 $/TWa in the comment.

Reply:
Oh, I see that was part of the long line nobody is going to read.  Still, comment up top with the rest would be better.